src/views/ApplicationViews.js

```diff
@@ -1,11 +1,15 @@
 import { Route, Routes } from "react-router-dom"
 import { Login } from "../components/auth/Login"
 import { Register } from "../components/auth/Register"
-import { Authorized } from "./Authorized"
+import { Authorized } from "./"
```

Comment:
I think the ./Authorized may have been deleted on line 4. Please add it the correct import logic back. Thank you